test/test_excerpt.rb

```diff
@@ -212,4 +212,56 @@ def do_render(document)
       assert_equal true, @excerpt.is_a?(Jekyll::Excerpt)
     end
   end
+
+  context "An excerpt with non-closed but valid Liquid block tag with whitespace control" do
```

Comment:
Rubocop is complaining about line length here. Not sure why this wasn't a problem in the original PR.

Reply:
We can always exclude this file from the LineLength Cop